src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -1,35 +1,63 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
 import { oneOf } from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { C_LUNAR_LIGHT } from '@bbc/psammead-styles/colours';
 import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 import { Helmet } from 'react-helmet';
-import AdSlot from './AdSlot';
 
 const LEADERBOARD_HEIGHT = '5.5rem';
 const LEADERBOARD_HEIGHT_GROUP_4_5 = '9rem';
+const MPU_HEIGHT = '9rem';
```

Comment:
This height is used to set a min-height for the MPU placeholder/grey-container surrounding the ad. 

I'm currently not sure what to set this value to be.

Reply:
@tochwill Do you know these values?

Reply:
I've changed this to use the value of the shorter MPU ad for now (which is 15.625rem). We can update this when get a final answer from MPU support! @DenisHdz